packages/lit-node-client-nodejs/src/lib/helpers/normalize-array.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Converts an ArrayLike object to a regular array.
+ *
+ * Context: the nodes will only accept a normal array type as a paramater due to serizalization issues with Uint8Array type. this loop below is to normalize the message to a basic array.
+ *
+ * @param toSign - The ArrayLike object to be converted.
+ * @returns The converted array.´
+ */
+export const normalizeArray = (toSign: ArrayLike<number>) => {
+  const arr = [];
+  for (let i = 0; i < toSign.length; i++) {
+    arr.push((toSign as Buffer)[i]);
```

Comment:
Thinking a cast to`Uint8Array` would be better here as to not keep using `Node` specific types like `Buffer` if we can avoid it.